services/assets/client.go

```diff
@@ -12,6 +12,16 @@ const (
 	AssetsURL = "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/"
 )
 
+func GetCoinInfo(coinId int) (info blockatlas.CoinInfo, err error) {
```

Comment:
this needs to take `token: String`

Data would be located here: https://github.com/trustwallet/assets/tree/master/blockchains/ethereum/assets/0x6B175474E89094C44Da98b954EedeAC495271d0F

Reply:
Once we add it.

Reply:
can we use:
`https://github.com/trustwallet/assets/tree/master/blockchains/ethereum/assets/0x6B175474E89094C44Da98b954EedeAC495271d0F/info/info.json`

instead
`https://github.com/trustwallet/assets/tree/master/blockchains/ethereum/assets/0x6B175474E89094C44Da98b954EedeAC495271d0F/info.json`

?

Reply:
I would rather move info.json out of info folder instead on assets repo, for consistency, otherwise too many folders